### DISCUSSION TITLE
[Unstable] Shifted railgun shells placement

### DISCUSSION THREAD

**Discussion by Minedeployder** [FakeFishGames/Barotrauma#14258] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

If you rotate anything that contains railgun shells, they are placed at wrong spot.

### Reproduction steps

1. Place railgun shell loader. 
2. Rotate to 180.
3. Load it with shells.
![Barotrauma Screenshot 2024 07 05 - 14 04 26 99](https://github.com/FakeFishGames/Barotrauma/assets/70541484/abe78d89-7010-4b62-8e44-4f1b99f7f943)
![Barotrauma Screenshot 2024 07 05 - 14 13 54 15](https://github.com/FakeFishGames/Barotrauma/assets/70541484/87bf1677-4f6c-4d3d-81a8-55182993b2a8)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Other

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has now been fixed, closing
